src/backend/mysql/query.rs

```diff
@@ -30,6 +30,22 @@ impl QueryBuilder for MysqlQueryBuilder {
         // MySQL doesn't support declaring materialization in SQL for with query.
     }
 
+    fn prepare_join_type(&self, join_type: &JoinType, sql: &mut dyn SqlWriter) {
+        write!(
+            sql,
+            "{}",
+            match join_type {
+                JoinType::Join => "JOIN",
+                JoinType::CrossJoin => "CROSS JOIN",
+                JoinType::InnerJoin => "INNER JOIN",
+                JoinType::LeftJoin => "LEFT JOIN",
+                JoinType::RightJoin => "RIGHT JOIN",
+                JoinType::FullOuterJoin => panic!("Mysql does not support FULL OUTER JOIN"),
+            }
```

Comment:
Personally, I think duplicating all other `JoinType` once more hurt maintainability. Maybe we can create a `prepare_join_type_common` method just like below? What's your take? @ikrivosheev 

https://github.com/SeaQL/sea-query/blob/cfe6bd2ef90fa7483d68364e8563c172de244947/src/backend/sqlite/query.rs#L23-L37

Reply:
@billy1624 yes, you are right! Thank you, fixed.